website/docs/atmo/usage/creating-runnables.md

```diff
@@ -22,5 +34,4 @@ name that will be used to call the Runnable in [Directive](../concepts/the-direc
 handlers, which are discussed next.
 
 Your Runnables will use the [Runnable API](../runnable-api/introduction.md) to access
-resources such as the network, files, and more.
-
+resources such as the network, files, and more.
```

Comment:
Since we are already changing this file, could you change lines 31-37 and remove the hard breaks where they are not needed (one line per paragraph)?

Reply:
👍